src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -39,7 +39,14 @@
         <v-col :cols="$vuetify.breakpoint.xs ? 12 : 3">
           <span>
             <v-icon class="grey--text text--lighten-1">mdi-account</v-icon>
-            Contact <strong>{{ currentDandiset.contact_person }}</strong>
+            <template
+              v-if="['', ','].includes(currentDandiset.contact_person.trim())"
```

Comment:
When a dandiset has a contact person with the name `','` that indicates that there is a contact person associated with it, but they do not have a first and last name. This happens when a user doesn't have a name set in their GitHub account. I wonder if it'd be better to fall back to their email if a name isn't set?

Reply:
What's the modern guidance on publishing people's email addresses? I've been allergic to it since 2003, on the assumption that evil bots are slurping up email addresses for spam. I wonder if the GitHub username would be a better fallback, and I wonder further if during the verification process (i.e., the user questionnaire flow) should require providing a name as well.

Reply:
ok, I will try to explore how to get the username

Reply:
> ok, I will try to explore how to get the username

:+1: 
just as a heads up, getting the github username would be something that happens on the API server (dandi-api). So, I think this PR could just be changed like this:

```suggestion
              v-if="!currentDandiset.contact_person"
```

And then it could be merged

Reply:
but assuming that `dandi-api` will return an empty string if there is no name, right now it returns ", ", so the if statement will always give `false`